components/organisms/EditAdmin/EditAdmin.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+import { useSession } from "next-auth/react";
+import styles from "./EditAdmin.module.css";
+import Link from "next/link";
+import { ArrowLeftIcon, Button } from "../../atoms";
+import IconDeleteBin6Fill from "../../atoms/icons/deleteIcon/DeleteIcon";
+import { useAppDispatch, useAppSelector } from "../../../store/store";
+import Swal from "sweetalert2";
+import { addAdmin } from "../../../services/user/addAdmin";
+import { useRouter } from "next/navigation";
+import AddCircleIcon from "../../atoms/icons/adminPanel/AddCircleIcon";
+import { fetchAdmins } from "../../../store/slices/allAdminsSlice";
+
+export const EditAdmin = () => {
+  const { data: session } = useSession();
+  const router = useRouter();
+  const allAdmins = useAppSelector((state) => state.allAdmins.admins);
+  const dispatch = useAppDispatch();
+
+  const handleDeleteAdmin = async (
+    adminEmail: string,
+    name: string,
+    lastName: string
+  ) => {
+    const admin = false;
+    const email = adminEmail;
+    const id = session?.user.uid || "";
+    const fullName =
+      name.charAt(0).toUpperCase() +
+      name.slice(1).toLowerCase() +
+      " " +
+      lastName.charAt(0).toUpperCase() +
+      lastName.slice(1).toLowerCase();
+
+    try {
+      const result = await Swal.fire({
+        title: "Estás seguro que deseas eliminar como administrador a:",
+        text: `${fullName} (${adminEmail})`,
+        icon: "warning",
+        showCancelButton: true,
+        confirmButtonText: "Sí, actualizar",
+        cancelButtonText: "Cancelar",
+        reverseButtons: true,
+      });
+
+      if (result.isConfirmed) {
+        const response = await addAdmin(email, id, admin);
+        if (response.status === 200) {
+          Swal.fire(
+            "Usuario eliminado!",
+            response.data.msg ||
+              "Rol de administrador actualizado exitosamente",
+            "success"
+          );
+          dispatch(fetchAdmins(id));
+
+          router.push("/dashboard/adminPanel/editAdmin");
+        } else {
+          Swal.fire(
+            "Error!",
+            response.data.message || "Administrador no eliminado correctamente",
```

Comment:
This message does not make sense to me… this admin wasn't eliminated successfully?